### PR TITLE
New tenant utilities script for installing new tenants

### PR DESCRIPTION
**Pull request by nick-funk**
## What does this PR do?

Adds a script under `utilities/tenant` that includes an installation script for new tenants on existing clusters that already have tenants.

## These changes will impact:

- [ ] commenters
- [ ] moderators
- [ ] admins
- [X] developers

## What changes to the GraphQL/Database Schema does this PR introduce?

None

## Does this PR introduce any new environment variables or feature flags?

No

## If any indexes were added, were they added to `INDEXES.md`?

No

## How do I test this PR?

- perform a development build of coral using `sh scripts/build-development.sh`
- `cd` into `utilities/tenant`
- run `npm i`
- perform a tenant install via ` npm run start -- --mongoURI mongodb://localhost:27017 --mongoDBName coral createTenant`
- follow the prompts for a new tenant
- check your Mongo db to see the new tenant was created
- change the domain of the new tenant to localhost
- modify the existing coral tenant to not be localhost
- start up Coral
- see that the new tenant is now being used instead of the old one

## Where any tests migrated to React Testing Library?

None

## How do we deploy this PR?

No special considerations.

### CODE REVIEW

File: server/src/core/server/models/tenant/helpers.ts [hunk 1]
```diff
@@ -10,29 +10,37 @@ import { AuthIntegrations } from "../settings";
...
+  bundle?: FluentBundle
+): Tenant["badges"] => {
+  return {
+    label: bundle ? translate(bundle, "Staff", "staff-label") : "Staff",
+    adminLabel: bundle ? translate(bundle, "Staff", "staff-label") : "Staff",
+    staffLabel: bundle ? translate(bundle, "Staff", "staff-label") : "Staff",
+    moderatorLabel: bundle
+      ? translate(bundle, "Staff", "staff-label")
+      : "Staff",
+    memberLabel: bundle ? translate(bundle, "Member", "member-label") : "Staff",
```
Comment (kabeaty) [inline server/src/core/server/models/tenant/helpers.ts]: Should this default to "Member" instead of "Staff" here in the case that `bundle` is false?

**Review by kabeaty — APPROVED** [review]
Works as expected and looks good! Just left one small comment.

File: server/src/core/server/models/tenant/helpers.ts (hunk 1 above)
Comment (nick-funk) [reply server/src/core/server/models/tenant/helpers.ts]: This is copied from the prior functionality. I haven't changed any of the logic. You may have discovered a bug? Or maybe this was done deliberately?

Comment (nick-funk) [reply server/src/core/server/models/tenant/helpers.ts]: Ah, looking closer, the default for all labels is just `Staff` if it's not available. This looks to be deliberately done in this way?

Comment (nick-funk) [reply server/src/core/server/models/tenant/helpers.ts]: You know what, this makes no sense to keep it as `Staff`, I'm changing it to `Member` as default.

Comment (kabeaty) [reply server/src/core/server/models/tenant/helpers.ts]: Thanks for looking into this and updating!

**Pull request merged**